bindings/java/nz/mega/sdk/MegaApiJava.java

```diff
@@ -214,6 +214,12 @@ void runCallback(Runnable runnable) {
     public final static int SEARCH_TARGET_ROOTNODE = MegaApi.SEARCH_TARGET_ROOTNODE;
     public final static int SEARCH_TARGET_ALL = MegaApi.SEARCH_TARGET_ALL;
 
+    public final static int STATE_NONE = MegaApi.STATE_NONE;
+    public final static int STATE_SYNCED = MegaApi.STATE_SYNCED;
+    public final static int STATE_PENDING = MegaApi.STATE_PENDING;
+    public final static int STATE_SYNCING = MegaApi.STATE_SYNCING;
+    public final static int STATE_IGNORED = MegaApi.STATE_IGNORED;
```

Comment:
these states refer to syncs, I don't think they should be included in bindings :thinking: